tests/openwisp2/sample_monitoring/migrations/0001_initial.py

```diff
@@ -131,7 +132,7 @@ class Migration(migrations.Migration):
                     'metric',
                     models.OneToOneField(
                         on_delete=django.db.models.deletion.CASCADE,
-                        to='sample_monitoring.Metric',
+                        to=swapper.get_model_name('monitoring', 'Metric'),
```

Comment:
This change has no effect on `sample_monitoring`, I only did it because if user tries to copy-paste the migration file in their own sample app(because people may copy the entire folder and start working on it), it would still work.